util/quicserver.c

```diff
@@ -12,6 +12,7 @@
  * by s_server and removed once we have full QUIC server support.
  */
 
+#include <stdio.h>
```

Comment:
Is this include still needed?

Reply:
I guess it's to get stderr

Reply:
Well, we use `stderr` so although it is probably included by some other dependency it is better to explicitly include it here too.